percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -139,6 +134,15 @@ get_sources(){
         git clean -xdf
         git checkout "$BRANCH"
     fi
+
+    if [ -z "${PSM_VER}" ]; then
+        PSM_VER=$(git describe --tags --abbrev=0 | sed 's/^psmdb-//' | awk -F '-' '{if ($2 ~ /^rc/) {print $0} else {print $1}}'
```

Comment:
There are no tags in our repository which starts with 'psmdb-' and have 'rc' part. So the logic handling 'rc' part is redundant.